src/system/SystemPacketBuffer.h

```diff
@@ -592,9 +575,25 @@ class DLL_EXPORT PacketBufferHandle
      *
      * @note This should be used ONLY by low-level code interfacing with LwIP.
      */
-    struct pbuf * GetLwIPpbuf() { return static_cast<struct pbuf *>(mBuffer); }
+    struct pbuf * UnsafeGetLwIPpbuf() { return static_cast<struct pbuf *>(mBuffer); }
```

Comment:
can we protect this method and make a list of friend classes allowed to call it?  is that worth the work?

Reply:
Yes, the only uses are in Inet:…EndPoint, so that shouldn't be difficult. (The same isn't true of UnsafeRelease(), unfortunately, since it's mostly used under src/platform/, so hiding it would be a problem for out-of-tree platforms.)

Reply:
Done with a task-specific subclass to set a pattern for disjoint exposures. No increase in code size.